pactman/verifier/verify.py

```diff
@@ -36,12 +36,12 @@ def __repr__(self):
     def __str__(self):
         return f"{self.pact.consumer} with request '{self.description}'"
 
-    def verify(self, service_url, setup_url):
+    def verify(self, service_url, setup_url, custom_headers=False):
```

Comment:
I've given this a shot. the _request_headers property isn't behaving as I'd expect when a setter is added to the property. I don't know why but suspect it might be something to do with the use of the @property decorator and the underscore in the property name (The decorated setter uses _<property_name>. ttps://www.programiz.com/python-programming/property. Using a double underscore results in the property not getting set, using a single results in an infinite loop)

I'm not sure I can justify the time to make this work when the optional params down the stack do the job, even if it is a little clunky.